pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -16,6 +16,7 @@
 , enableThreads ? true
 , enableSmpSupport ? true
 , enableKernelPoll ? true
+, ssl ? openssl,
```

```suggestion
, ssl ? openssl
```
We put the commas in front.